### DISCUSSION TITLE
Does meow support visual-block select and multi cursor editing like vim？

### DISCUSSION THREAD

**Discussion by G4RASU** [meow-edit/meow#522] [Q&A]
Here is the [example](https://stackoverflow.com/questions/11784408/vim-multiline-editing-like-in-sublimetext)
 Oh, I recently switched to emacs from vim. Don't know if this meets the idea of this package, any other package provides similar function works for me.

**Comment by ideasman42** [comment]
This seems to be have closed without an answer.
Is this supported.

**Reply by brongulus** [reply]
I use `rectange-mark-mode` and its related functions for this.

**Reply by underside** [reply]
Multi-line editing: 
Select text block -> "G" -> edit -> "G"

It's in Meow tutor (M-x -> meow-tutor ).
